changelog/changes/public-maybe-banner.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Add IP address classification functions"
```

Comment:
Ah, minor nitpick: Never choose the PR title as the changelog title. Almost never works. (I should really change the bot to not prefill this)

Reply:
Good call! Fixed in 5eb33f3.